druid/src/core.rs

```diff
@@ -1014,14 +1015,24 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                 (Some(_), Some(_)) => {}
             }
         }
-
-        for (_, host) in &self.state.sub_window_hosts {
-            let cloned: T = (*data).clone();
-            let command = Command::new(SUB_WINDOW_PARENT_TO_HOST, Box::new(cloned), *host);
-            ctx.submit_command(command);
-        }
-
         let prev_env = self.env.as_ref().filter(|p| !p.same(env));
+        let env_changed = prev_env.is_some();
+        let data_changed = self.old_data.as_ref().filter(|p| !p.same(data)).is_some();
```

Comment:
won't this be incorrect in the case where there is no old data?

Reply:
For some reason I thought that couldn't happen but not sure why...